general_rules/Robots-OPL.tex

```diff
@@ -47,4 +50,11 @@ \subsubsection{Audio output plug}\label{rule:roobt_audio_out}
 		\item It is the responsibility of the teams to plug in the transmitter before a test, to check the sound system, and to hand over the transmitter to next team.
 		\item Do not rely on the sound system! For fail-safe operation and interacting with operators make sure that the sound system is not needed, e.g., by having additional speakers directly on the robot.
 \end{compactitem}
-\end{enumerate}
\ No newline at end of file
+\end{enumerate}
+
+
+
+
+\subsubsection{Appearance}
+\label{rule:robots_appearance}
```

Comment:
I remember that Pumas (@kyordhel's former team) used AR-markers for closed-controlled manipulation, but I remember it more recently (pretty sure it was Netherlands 2013). I did not know that TC back in those days asked teams to remove them.

In any case, I really don't mind if the stickers are on the robot's body in OPL, since the philosophy in that league is that they can do whatever they want with the body of the robot (as long as they don't modify the arena in any way). However, in both standard platforms, putting a sticker that isn't for promotional/decorative use and obviously for ease of hand tracking, is walking the line of not being appropriate:

CON PUTTING THE STICKER: the philosophy behind the standard platform leagues is that the software solutions that they come up with can be used in any instance of that hardware platform. If they move that solution to another robot that doesn't have the sticker, their software doesn't work anymore.

PRO PUTTING THE STICKER: it's just a bleeping sticker, which is easily installable.

Ugh... I hate these kinds of issues.

Compromise: let them use stickers, but it is up for the TC during the competition if they are within the "gentleman rule" in the rule book.